src/main/java/ichttt/mods/mcpaint/common/capability/CapabilityPaintable.java

```diff
@@ -49,4 +58,9 @@ public static void readFromNBT(IPaintable instance, CompoundTag compound) {
         }
         instance.setData(scaleFactor, pictureData, null, null);
     }
+
+    @NotNull
+    protected static <T> Capability<T> getCapability(CapabilityToken<T> type) {
+        return CapabilityManager.get(type);
+    }
```

Comment:
I don't think that this helper method is useful, could just call CapabilityManager.get in line 16